#! /bin/sh /usr/share/dpatch/dpatch-run

## 06_default_svdrp_port_0.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This sets the default svdrp-port to 0, which means that
## DP: SVDRP is disabled by default unless someone specifies
## DP: another port with the --port option when starting vdr

@DPATCH@
diff -urNad vdr-1.3.38/vdr.c /tmp/dpep.gHaEyx/vdr-1.3.38/vdr.c
--- vdr-1.3.38/vdr.c	2006-01-08 20:17:04.526755250 +0100
+++ /tmp/dpep.gHaEyx/vdr-1.3.38/vdr.c	2006-01-08 20:17:22.595884500 +0100
@@ -158,7 +158,7 @@
   // Command line options:
 
 #define DEFAULTVDRUSER   "vdr"
-#define DEFAULTSVDRPPORT 2001
+#define DEFAULTSVDRPPORT 0
 #define DEFAULTWATCHDOG     0 // seconds
 #define DEFAULTPLUGINDIR PLUGINDIR
 #define DEFAULTEPGDATAFILENAME "epg.data"
